docs/_articles/en/einstein/einstein-faq.md

```diff
@@ -17,7 +17,7 @@ See [Trusted AI from Salesforce](https://www.salesforceairesearch.com/trusted-ai
 
 **Will my code ever be shared outside of my development environment?**
```

Comment:
I think this question should be stated as:
"Will my code ever be shared outside of Salesforce?"

Reply:
agreed. Updating.